crates/cli/src/opts/build/core.rs

```diff
@@ -126,11 +126,6 @@ pub struct CoreBuildArgs {
     #[serde(skip)]
     pub skip: Option<Vec<SkipBuildFilter>>,
 
-    /// Build source files from specified paths.
-    #[arg(long, short, num_args(0..))]
-    #[serde(skip)]
-    pub paths: Option<Vec<PathBuf>>,
```

Comment:
ah this was in core args, which means it was present in all sub commands, i don't think moving it to pos args is equivalent

Reply:
it is only used and respected in `BuildArgs::run`, so it belongs to `BuildArgs` anyway